rest-service/manager_rest/rest/resources_v3/tenants.py

```diff
@@ -128,6 +128,34 @@ def put(self, multi_tenancy):
         )
 
     @rest_decorators.exceptions_handled
+    @authorize('tenant_update_user', get_tenant_from='data')
+    @rest_decorators.marshal_with(TenantResponse)
+    @rest_decorators.no_external_authenticator('update user in tenant')
+    def patch(self, multi_tenancy):
+        """Update role in user tenant association."""
+        request_dict = rest_utils.get_json_and_verify_params(
+            {
+                'tenant_name': {
+                    'type': unicode,
+                },
+                'username': {
+                    'type': unicode,
+                },
+                'role': {
+                    'type': unicode,
+                },
+            },
+        )
+        rest_utils.validate_inputs(request_dict)
+        role_name = request_dict['role']
+        rest_utils.validate_role_name(role_name)
```

Comment:
Yariv changed the "validate_role_name" with "verify_role"
seems to update is function, since it should get both system and tenant roles..